src/System.Private.CoreLib/shared/System/Globalization/CultureInfo.cs

```diff
@@ -588,7 +589,7 @@ public virtual CultureInfo Parent
             {
                 if (_parent == null)
                 {
-                    CultureInfo culture = null;
+                    CultureInfo? culture = null;
```

Comment:
Better to remove assigment of null; and change the code below to `CreateCultureInfoNoThrow(...) ?? InvariantCulture;`